src/tiled/automappingutils.cpp

```diff
@@ -73,7 +77,8 @@ const QList<MapObject*> objectsInRegion(ObjectGroup *layer,
         // QRegion::intersects() returns false for empty regions even if they are
         // contained within the region, so we also check for containment of the
         // top left to include the case of zero size objects.
-        if (where.intersects(rect) || where.contains(rect.topLeft()))
+        // (Also, for Tile objects we need to check the bottom left)
+        if (where.intersects(rect) || where.contains(rect.topLeft()) || where.contains(rect.bottomLeft()))
```

Comment:
It should not matter whether you check the top left or bottom left for tile objects, since their size is 0.